setup.cfg

```diff
@@ -35,10 +35,10 @@ install_requires =
     packaging
     tornado>=6.1.0
     jupyter_core
-    jupyterlab_server~=2.10
-    jupyter_server~=1.4
+    jupyterlab_server>=2.10
+    jupyter_server>=1.8
```

Comment:
Do we want to allow future major versions to be installed with older JupyterLab?

Reply:
This comes from the advice in a [blog post](https://iscinumpy.dev/post/bound-version-constraints/) about dependency handling in Python, cf https://github.com/jupyter-server/jupyter_server/issues/699.

Reply:
I take your point that JupyterLab is an application, and tightly coupled to its server, I put the version constraints back in for those.

Reply:
Thanks @blink1073